src/main/resources/bidder-config/imds.yaml

```diff
@@ -23,6 +23,7 @@ adapters:
         support-cors: true
         uid-macro: '[USER_ID]'
       redirect:
+        # IMDS does not support syncing outside of the US/Canada
```

Comment:
This might be better located higher in the YAML. This comment applies to BOTH the `redirect` and `iframe` syncs.

Reply:
good call